### DISCUSSION TITLE
select option default value

### DISCUSSION THREAD

**Discussion by idc77** [react-hook-form/react-hook-form#12343] [Q&A]
I'm sorry if this has already been answered, there are 12 pages of select search results. And 6 pages with the title searched, few go into that direction but nothing useful.

There's this chicken and egg problem.

```html
<select>
  <option selected disabled></option>
</select>
```
If you do this React complains and annoys with a warning, that I should set a default value or use defaultValue.
```
Warning: Use the `defaultValue` or `value` props on <select> instead of setting `selected` on <option>. 
```

But,
I'd like the select element to have a title, and I'd like validation required to be working.
I tried
```js
{required: "A message", validation: (value, Formvalue) => value !== "DEFAULT"}
```
When I click the submit button, nothing happens.

How can I have my cake and eat it too?
Can I disable the warning?

**Comment by wilsonadenuga** [comment]
I am not sure i get the full picture from your description, maybe add the full form here?
